# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436967528 14400
#      Wed Jul 15 09:38:48 2015 -0400
# Node ID 83d08eb76182561879667a368ab74598457e2aed
# Parent  4847e3dd8e6abb85d1d1f7144bf1e3dc42372ef2
* doc/module.mk (doxyhtml-maintainer-clean): Also remove .db file.

diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -36,16 +36,17 @@ EXTRA_DIST += \
   doc/doxyhtml/Doxyfile.in \
   doc/doxyhtml/README
 
 doxyhtml: doc/doxyhtml/$(octave_dirstamp)
 	doxygen doc/doxyhtml/Doxyfile
 
 doxyhtml-maintainer-clean:
 	cd doc/doxyhtml && rm -rf `ls | $(GREP) -v Doxyfile.in`
+	rm -f doc/doxygen_sqlite3.db
 
 DIRSTAMP_FILES += doc/doxyhtml/$(octave_dirstamp)
 
 GRAPH_PROP_TEXI_SRC = \
   doc/interpreter/plot-axesproperties.texi \
   doc/interpreter/plot-figureproperties.texi \
   doc/interpreter/plot-imageproperties.texi \
   doc/interpreter/plot-lineproperties.texi \
